corehq/apps/reports/standard/forms/filters.py

```diff
@@ -74,7 +75,4 @@ def filter_context(self):
     @classmethod
     def get_filter_toggle(cls, request):
         filter_ = request.GET.getlist(cls.slug, None)
-        if filter_:
-            return SubmissionErrorType.use_filter(filter_)
-        else:
-            return SubmissionErrorType.use_error_defaults()
+        return SubmissionErrorType.use_filter(filter_)
```

Comment:
This is getting rid of the default selected options, isn't it? Why?

Reply:
The defaults looked arbitrary (it had the `Form with Errors` and `Generic Error` buttons default to being selected), I figured it was legacy code that didn't make sense anymore, so now it just default to every button being deselected

Reply:
@djmore9 should there be a default value? We're talking about the "Submission Type" filter on the [Raw Forms, Errors & Duplicates report](https://www.commcarehq.org/a/bosco/reports/submit_errors/)